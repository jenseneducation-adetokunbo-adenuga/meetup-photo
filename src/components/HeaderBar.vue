<template>
  <v-container>
    <v-app-bar app>
      <!-- -->
    </v-app-bar>
  </v-container>
</template>

<script>
export default {
  name: "HeaderBar",

  data: () => ({
    //
  }),
};
</script>
