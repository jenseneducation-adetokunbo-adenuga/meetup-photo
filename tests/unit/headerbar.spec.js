import HeaderBar from "@/components/HeaderBar";
import { createLocalVue, mount } from "@vue/test-utils";
import Vuetify from "vuetify";

describe("HeaderBar.vue", () => {
  const localVue = createLocalVue();
  let vuetify;

  beforeEach(() => {
    vuetify = new Vuetify();
  });

  it("should have a custom title and match snapshot", () => {
    const wrapper = mount(HeaderBar, {
      localVue,
      vuetify,
      propsData: { title: "Foobar" },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
