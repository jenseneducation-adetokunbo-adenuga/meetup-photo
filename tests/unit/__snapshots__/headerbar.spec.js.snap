// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderBar.vue should have a custom title and match snapshot 1`] = `
<v-container title="Foobar">
  <v-app-bar app=""></v-app-bar>
</v-container>
`;
